### ISSUE TITLE
ABS Cannot Match Authors or Books from Audible

### ISSUE THREAD

**Issue by petenun** [mikiher/audiobookshelf-windows#22]
I just noticed today that Audiobookshelf is not finding any results when looking up authors or when matching books against Audible. Matching from iTunes and Google Books, and Open Library seem to work. This is the first I've encountered the problem, so it could be that external lookup services are down. I'm running ABS v2.13.3 on Windows Server 2019.

Here's an example error when looking up author:

> 2024-09-06 21:04:26.426		INFO		[Audnexus] Searching for author "https://api.audnex.us/authors?name=daphne+du+maurier"
> 
> 2024-09-06 21:04:26.427		INFO		[Audnexus] Searching for author "https://api.audnex.us/authors?name=the+rendezvous+and+other+stories"
> 
> 2024-09-06 21:04:26.434		ERROR		[Audnexus] Author ASIN request failed for daphne du maurier Error: getaddrinfo ENOTFOUND api.audnex.us
> 

==========

And when looking up a book from Audible:

> 2024-09-06 18:03:14.944		ERROR		[Audible] ASIN search error Error: getaddrinfo ENOTFOUND api.audnex.us

I looked, but it does not appear that anything is written to Windows Application Logs.

**Comment by petenun** [comment]
It looks like audnex.us disappeared from DNS completely - it just can't be found. However, I found a post on https://github.com/advplyr/audiobookshelf/issues/3385 that notes that the IP addresses still work though. Since DNS isn't working, I had to modify my windows hosts file (c:\windows\system32\drivers\etc\hosts) in Notepad and add the following two lines. It's not a perfect work-around, but it appears to be functional.

Append these lines to the Hosts file:

104.21.44.87	audnex.us
104.21.44.87	api.audnex.us
